src/index.html

```diff
@@ -1,13 +1,368 @@
+              </div>
+            </div>
+          </div>
+          <div class="details__photos">
+            <div class="grid grid--desktop grid--tablet">
+              <div class="grid__item grid__item1-4-d grid__item1-2-t">
+                <div class="details__photo-container">
+                  <img
+                    src="./images/details/5.png"
+                    alt="Place 3"
+                    class="details__photo"
+                  >
+                </div>
+              </div>
+              <div class="grid__item grid__item5-12-d grid__item3-6-t">
+                <div class="details__photo-container">
+                  <img
+                    src="./images/details/6.png"
+                    alt="Place 3"
+                    class="details__photo"
+                    >
+                </div>
+              </div>
+            </div>
+            <div class="details__description">
+              <h3 class="details__description-title">
+                Hydraulic disc brakes & Lightweight
+              </h3>
+              <div class="details__description-info">
+                Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="grid--desktop grid--tablet">
+          <div class="grid__item grid__item5-8-d grid__item2-5-t">
+            <label>
+              <a href="#" class="button">
+                Explore
+              </a>
+            </label>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+          <div class="grid--desktop grid--tablet">
+            <div class="grid__item grid__item1-5-d grid__item1-3-t">
+              <label>
+                <form action="#" method="get">
+                  <input
+                  type="text"
+                  name="name"
+                  class="form-field contact-us__field"
+                  placeholder="Name"
+                  required
+                  >
+                  <input
+                  type="email"
+                  name="email"
+                  class="form-field contact-us__field"
+                  placeholder="Email"
+                  required
+                  >
+                  <textarea
+                    name="message"
+                    class="form-field form-field--textarea contact-us__field"
+                    placeholder="Message"
+                  ></textarea>
+                  <button class="button">
+                    Send
+                  </button>
```

Comment:
add type submit. And use [preventDefault](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) to stop updating page (look on example, you need the same just event will be submit instead of click),  and [reset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) method.

Go to qna if hard to do it